### DISCUSSION TITLE
A question about controlling computational resource

### DISCUSSION THREAD

**Discussion by Tomcxf** [nextflow-io/nextflow#4252] [Q&A]
Hi!
I want to compare two pipeline whil on is written in nextflow and the other is snakemake by testing the same data.

So I need to make the resource used by two the same.

I choose to allocate 8 cpus to snakemake. From the shell command 'top', it really show that 8 lines and each of their %CPU is 100.

However, when I set param for nextflow in its local.config, although there are 8 lines, each %CPU is ~800, which means using 64cpus- the maxium of my computation resource.

So how can I allocate resource to make two the same?

Thank you!

**Comment by bentsherman** [comment]
Can you share your pipeline script and config file? I suspect that your pipeline launched 8 tasks with 8 cpus each, when I'm guessing you wanted 8 tasks with 1 cpu each, or vice versa.
